motor_data_update_rate: {
  sgl: -2.,
  sgr: -2.,
  pwm: -1.,
  flags: 0.5,
  current: -1.,
  voltage: -2.,
  temperature: 1.,
  can_num_received: 0.1,
  can_num_transmitted: 0.1,
  slow_data: -2.,
  can_error_counters: 0.1,
  pterm: -1.,
  iterm: -1.,
  dterm: -1.
  }
